Remove langchain (#22)
glific/Open-LLM: README.md

> We use Python 3.9 pipenv for managing dependencies and create the virtual environment.

Reviewer: ```suggestion
This project uses Python 3.9 and virtual environments:
```

> Create a virtualenv using pipenv:

Reviewer: ```suggestion
Create a virtualenv:
```
